packages/design-system/src/components/Table/TableHeaderCell.jsx

```diff
@@ -9,14 +10,18 @@ export const TableHeaderCell = ({
   scope,
   stackedTitle,
   stackedClassName,
-  width,
   ...others
 }) => {
-  const classes = classNames(
-    'ds-c-table__header',
-    className,
-    width ? 'ds-c-table__header--width-' + width : null
-  );
+  const tableStackableContext = useContext(TableContext);
+  if (process.env.NODE_ENV !== 'production') {
+    if (tableStackableContext && !id) {
+      console.warn(
+        `Please provide an 'id' prop for stackable table, it is a required prop for screen readers to create association between the heading id and data cells.`
```

Comment:
Similar feedback as above, we want to have relatively consistent messaging, you can also see examples of this in `HelpDrawer.jsx`